C/CImGuiPack/build_tarballs.jl

```diff
@@ -3,20 +3,20 @@
 using BinaryBuilder
 
 name = "CImGuiPack"
-version = v"0.1.2"
+version = v"0.1.3"
 
 # Collection of sources required to build CImGuiPack
 sources = [
     GitSource("https://github.com/JuliaImGui/cimgui-pack.git",
-              "ecc93265e2f0f916462a2bef427f5f6b024ad6b2")
+              "af96d7d92c8b430af35e4b87c1260b234ccf9b5a")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/cimgui-pack
 git submodule update --init --recursive
 mkdir build && cd build
-cmake .. -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+cmake .. -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DImDrawIdx='unsigned int'
```

Comment:
Adding `-DImDrawIdx='unsigned int'` in the CMake command like this doesn't automatically append it to the CFLAGS/CXXFLAGS that are used by the compiler, this just passes it into CMake. So therefore you aren't actually compiling your source file with this flag currently. You can see that CMake is just ignoring it if you look into the build logs and see:

```
-- Configuring done

CMake Warning:
  Manually-specified variables were not used by the project:

    ImDrawIdx

Build files have been written to: /workspace/srcdir/cimgui-pack/build
```

You should either add another `add_definitions` statement in your CMakeLists.txt (since I guess you control it and you aren't using the upstream one), or manually set it on the command line by doing `-DCMAKE_CXX_FLAGS="-DImDrawIdx='unsigned int" -DCMAKE_C_FLAGS="-DImDrawIdx='unsigned int"` (I would say the first is probably the best way).